### PR TITLE
Replace reload4j with log4j2

### PR DESCRIPTION
**Pull request by PeterSuna**

### CODE REVIEW

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-helm/helm/lighty-rcgnmi-app-helm/templates/configmaps.yaml [hunk 1]
```diff
@@ -62,11 +62,20 @@ data:
...
-  log4j.properties: |
-    log4j.rootLogger=INFO, STDOUT
-    log4j.appender.STDOUT=org.apache.log4j.ConsoleAppender
-    log4j.appender.STDOUT.layout=org.apache.log4j.PatternLayout
-    log4j.appender.STDOUT.layout.ConversionPattern=%5p %d{HH:mm:ss} [%c] (%F:%L) - %m%n
+  log4j2.xml: |
+    <Configuration status="INFO">
+        <Appenders>
+            <Console name="ConsoleAppender" target="SYSTEM_OUT">
+                <PatternLayout pattern="%5p [%t] (%F:%L) - %m%n" />
```
Comment (ihrasko) [inline lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-helm/helm/lighty-rcgnmi-app-helm/templates/configmaps.yaml:69]: Originally, there was a different pattern.

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/templates/configmaps.yaml [hunk 2]
```diff
@@ -98,11 +98,20 @@ data:
...
-  log4j.properties: |
-    log4j.rootLogger=INFO, STDOUT
-    log4j.appender.STDOUT=org.apache.log4j.ConsoleAppender
-    log4j.appender.STDOUT.layout=org.apache.log4j.PatternLayout
-    log4j.appender.STDOUT.layout.ConversionPattern=%5p %d{HH:mm:ss} [%c] (%F:%L) - %m%n
+  log4j2.xml: |
+    <Configuration status="INFO">
+        <Appenders>
+            <Console name="ConsoleAppender" target="SYSTEM_OUT">
+                <PatternLayout pattern="%5p [%t] (%F:%L) - %m%n" />
```
Comment (ihrasko) [inline lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/templates/configmaps.yaml:105]: Originally, there was a different pattern.

**Review by ihrasko — CHANGES_REQUESTED** [review]

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-helm/helm/lighty-rcgnmi-app-helm/templates/configmaps.yaml (hunk 1 above)
Comment (PeterSuna) [reply lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-helm/helm/lighty-rcgnmi-app-helm/templates/configmaps.yaml]: Fixed

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/templates/configmaps.yaml (hunk 2 above)
Comment (PeterSuna) [reply lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/templates/configmaps.yaml]: Fixed

**Review by ihrasko — APPROVED** [review]

**Pull request merged**